save-core/build.gradle.kts

```diff
@@ -77,29 +77,30 @@ tasks.withType<KotlinCompile<*>>().forEach {
 }
 
 tasks.register<Download>("downloadTestResources") {
-    src(listOf(
-        Versions.IntegrationTest.ktlintLink,
-        Versions.IntegrationTest.diktatLink,
-    ))
+    src(Versions.IntegrationTest.ktlintLink)
+    overwrite(false)
+    onlyIfModified(true)
     dest("../examples/kotlin-diktat")
-    doLast {
-        Files.move(
-            file("../examples/kotlin-diktat/diktat-${Versions.IntegrationTest.diktat}.jar").toPath(),
-            file("../examples/kotlin-diktat/diktat.jar").toPath(),
-            StandardCopyOption.REPLACE_EXISTING
-        )
-    }
+    dependsOn("downloadTestResources2")
 }
 
-val cleanupTask = tasks.register("cleanupTestResources") {
-    this.dependsOn(":save-cli:jvmTest")
-    mustRunAfter(tasks.withType<Test>())
-    doFirst {
-        file("../examples/kotlin-diktat/ktlint").delete()
-        file("../examples/kotlin-diktat/diktat.jar").delete()
-    }
+tasks.register<Download>("downloadTestResources2") {
+    src(Versions.IntegrationTest.diktatLink)
```

Comment:
does this download task just move a `.jar` file or is this downloading from the internet?

Reply:
if we're downloading jars from the internet, it might be better to use mavencentral so it can be kept across builds